src/index.js

```diff
@@ -30,26 +31,25 @@ const editTask = (desc, index) => {
   addToLocalStorage();
 };
 
-const setState = (tasks, checkbox, index) => {
-  if (checkbox.checked === true) {
-    tasks[index].checked = true;
-  } else {
-    tasks[index].checked = false;
-  }
-};
-
 const displayTasks = () => {
   taskWrapper.innerHTML = '';
+  taskWrapper.classList.add('drag-list');
```

Comment:
- [ ] the to-do tasks are still not displayed on the screen of the user, this issue originates from the fact that you are looking for `.to-dos` class in the `index.js` however when running `npm run build` and checking the `index.html` in the dist folder there is not tag with this class name. therefore it is not able to find anything.